### PR TITLE
UNI-357

### PR DESCRIPTION
**Pull request by marionleborgne**
Fixes https://jira.numenta.com/browse/UNI-357 and linked to @ywcui1990 's PR: https://github.com/numenta/numenta-apps/pull/717

### CODE REVIEW

File: unicorn/tests/py/integration/compatibility_test/model_runner_2_compatibility_test.py [hunk 1]
```diff
@@ -160,17 +164,60 @@ def _testModelRunner(self, name, inputSpec, aggSpec, modelSpec):
...
-        trueDataValue = float(trueResult[1])
-        trueAnomalyLikelihood = float(trueResult[2])
-        self.assertAlmostEqual(outputRecord[1], trueDataValue, places=7)
-        self.assertAlmostEqual(outputRecord[2], trueAnomalyLikelihood, places=7)
+      numFalsePositiveComputed = self._checkForFalsePositives(
+        nabLabels, computedDetection, anomalyWindow, probationaryPeriod)
+      numFalsePositiveNAB= self._checkForFalsePositives(
+        nabLabels, nabDetection, anomalyWindow, probationaryPeriod)
 
+      print "Dataset: ", name
```
Comment (brev) [inline unicorn/tests/py/integration/compatibility_test/model_runner_2_compatibility_test.py]: @marionleborgne Are these `print`s just debugging statements?

File: unicorn/README.md [hunk 2]
```diff
@@ -200,10 +200,25 @@ npm install --no-optional
...
+#### Install
 Execute the following command after making changes to `unicorn_backend` code:
 ```shell
 npm run install:backend
 ```
+IMPORTANT: run this if you updated the python code in `py/unicorn_backend` and you 
+want these changes to take effect in the portable python distribution run 
+by the electron app.
+
+#### Test
```
Comment (brev) [inline unicorn/README.md]: @marionleborgne I'd expect the Test info to be lower in the Testing section

File: unicorn/package.json [hunk 3]
```diff
@@ -43,13 +43,15 @@
     "lint": "eslint --ext=js --ext=jsx js/ tests/js/",
     "prepare": "npm run lint && webpack",
     "pretest": "npm run prepare",
-    "install:backend": "python py/setup.py install",
+    "install:backend": "rm -rf node_modules/portable_python_darwin/lib/python2.7/site-packages/unicorn_backend* && python py/setup.py install",
```
Comment (brev) [inline unicorn/package.json]: @marionleborgne perhaps move the `rm` to a `clean:backend` run-script? And then probably call that from `preinstall:backend`

**Comment by brev** [comment]
something wrong with my home box, I'll try and test again in the morning from the office. thanks

**Comment by marionleborgne** [comment]
Thanks @brev. I addressed your comments.

**Pull request merged**